Day-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,43 @@
 /* All styles for your header Components live here */
+.container-header{
+    height: 50px;
+    padding: 10px;
+    display:flex;
+}
+
+.thumb-wrapper{
+    height:50px;
+    padding:0.3em;
+}
+
+.thumb-img{
+    height:100%;
+}
+
+.time-stamp{
+    font-weight: 300;
+    padding:5px;
+}
+
+.header-title{
+    font-size:14px;
+    font-weight:bold;
+    padding: 0 0.3em;
+}
+
+   .span .header-title{
+    opacity: 0.5;
+}
+
+.header-title-wrapper{
+    flex-direction: column;
+    margin-top: 0.6em;
+}
+
+.header-content{
+    /* font-size: 13px; */
```

Comment:
Commented code should not be pushed to GitHub.

Reply:
That’s fine and all but why are you telling me this 4 years later? 🤣